src/Symfony/Component/Validator/Constraints/IbanValidator.php

```diff
@@ -57,7 +57,7 @@ class IbanValidator extends ConstraintValidator
         'CH' => 'CH\d{2}\d{5}[\dA-Z]{12}', // Switzerland
         'CI' => 'CI\d{2}[A-Z]{1}\d{23}', // Ivory Coast
         'CM' => 'CM\d{2}\d{23}', // Cameron
-        'CR' => 'CR\d{2}\d{3}\d{14}', // Costa Rica
+        'CR' => 'CR\d{2}?0?\d{3}\d{14}', // Costa Rica
```

Comment:
From Wikipedia the `0` is required if I'm not misreading. Therefore `CR\d{2}0\d{3}\d{14}` ?

Reply:
Yes, but I didn't wanted to introduce BC - so without 0 will also work (like before).

Reply:
https://bank.codes/iban/structure/costa-rica/